airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/up_to_date/commands.py

```diff
@@ -42,18 +42,23 @@
     default=False,
     help="Create a pull request.",
 )
-
+@click.option(
+    "--no-bump",
```

Comment:
nit: the negative flag is a little confusing. why not `--bump`?

Reply:
I had the same thought, but honestly, good enough for an internal tool for now.